src/client/features/interactions/filter-menu.js

```diff
@@ -13,9 +13,8 @@ class InteractionsFilterMenu extends React.Component {
         onClick: () => props.filterUpdate(type)
       },
       [
-        h('div',{className:classNames(type,'interaction-filter-legend')}),
+        h('div',{className:classNames('interaction-filter-legend',{[type]:!clicked})}),
```

Comment:
what does ```[type]: !clicked``` mean?

Reply:
The [type] is the class that will be applied based off of which button it is and  [type]: !clicked only applies that class if the filter is active.